dll/win32/msgina/lang/ro-RO.rc

```diff
@@ -54,76 +54,76 @@ CAPTION "Securitate"
 FONT 8, "MS Shell Dlg", 400, 0, 1
 BEGIN
     CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
-    GROUPBOX "Informații de autentificare", IDC_STATIC, 7, 50, 261, 52
-    LTEXT "<Message>", IDC_SECURITY_MESSAGE, 15, 62, 245, 18
-    LTEXT "<Logon date>", IDC_SECURITY_LOGONDATE, 15, 83, 245, 8
-    CTEXT "Ce acțiune preferați?", IDC_STATIC, 10, 108, 255, 8
-    PUSHBUTTON "&Blochează calculatorul", IDC_SECURITY_LOCK, 7, 124, 80, 14
-    PUSHBUTTON "&Deautentifică-mă…", IDC_SECURITY_LOGOFF, 98, 124, 80, 14
-    PUSHBUTTON "Închid&e calculatorul…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
-    PUSHBUTTON "Schim&bă parola…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
-    PUSHBUTTON "&Gestionar de activități", IDC_SECURITY_TASKMGR, 98, 143, 80, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 188, 143, 80, 14
+    GROUPBOX "Informaţii de Log on", IDC_STATIC, 7, 50, 261, 52
+    LTEXT "<Mesaj>", IDC_LOCKED_MESSAGE, 15, 62, 245, 18
```

Comment:
you changed IDC_SECURITY_MESSAGE to IDC_LOCKED_MESSAGE here.

Reply:
I will change it back. I didn't pay attention.

Reply:
Fixed!